src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import { requestPasswordChange } from '../../actions/userActions';
+import Input from '../ui-components/Input';
+import Button from "../ui-components/Button";
+
+import routes from '../../constants/routes';
+
+import './ForgotPassword.scss';
+
+const START = 'START';
+const COMPLETE = 'COMPLETE';
+const ERROR = 'ERROR';
+
+const initialState = {
+  request: START,
+  email: '',
+  error: false,
+};
+
+class ForgotPassword extends React.Component {
+  state = { ...initialState };
+
+  validateEmail(email) {
+    const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
We should put this in a util to be used in the other forms as well.

Reply:
moved 👍